app/jinja_filters.py

```diff
@@ -574,19 +574,38 @@ def map_summary_item_config(
     edit_link_text: str,
     edit_link_aria_label: str,
     calculated_question: SelectFieldBase._Option,
+    remove_link_text: Optional[str],
+    remove_link_aria_label: Optional[str],
+    icon: Optional[str] = None,
 ) -> list[SummaryRow]:
 
-    rows = [
-        SummaryRow(
-            block["question"],
-            summary_type,
-            answers_are_editable,
-            no_answer_provided,
-            edit_link_text,
-            edit_link_aria_label,
-        )
-        for block in group["blocks"]
-    ]
+    rows = []
+
+    for block in group["blocks"]:
+        if block.get("question"):
+            rows.append(
+                SummaryRow(
+                    block["question"],
+                    summary_type,
```

Comment:
Do we need to update the type for `summary_type`? SummaryRow expects a `SelectFieldBase._Option` but we only seem to pass it as a `str`

Reply:
Yeah, just changed it to field base option, thanks!